### PR TITLE
Update test case P01 multiAZ to take account of the changes in the perf test suite

### PR DESCRIPTION
**Pull request by austincunningham**
# Description

https://issues.redhat.com/browse/MGDAPI-1535
Update test case P01 multiAZ to take account of the changes in the perf test suite
Add a fix for `./scripts/alerts-during-perf-testing.sh`
- change default to rhoam
- add an output for the script when alerts are firing

## Type of change

- [x] update test case

## Verification of script
- run the script `./scripts/alerts-during-perf-testing.sh`
- Trigger an alert 
   - go to the redhat-rhoam-middleware-monitoring-operator ns
   - scale down the application-monitoring-operator deployment
 - wait for about 5 min for the alert to trigger
 - you should see the script output change to show the alerts
 ```bash
Fri 16 Apr 2021 13:56:38 IST
Only alert firing is DeadMansSwitch
Fri 16 Apr 2021 13:56:40 IST
Only alert firing is DeadMansSwitch
Fri 16 Apr 2021 13:56:41 IST
Only alert firing is DeadMansSwitch
Fri 16 Apr 2021 13:56:42 IST
Only alert firing is DeadMansSwitch
Fri 16 Apr 2021 13:56:43 IST
============================================================================
Following alerts are Firing at :
Fri 16 Apr 2021 13:56:44 IST
"DeadMansSwitch","firing","2021-04-16T11:39:05.130123877Z"
"RHMIMiddlewareMonitoringOperatorApplicationMonitoringMetricsServiceEndpointDown","firing","2021-04-16T12:51:43.435642699Z"
============================================================================
Following alerts are Pending :
Fri 16 Apr 2021 13:56:44 IST
"MiddlewareMonitoringPodCount","pending","2021-04-16T12:55:22.883756212Z"
"RHMIMiddlewareMonitoringOperatorApplicationMonitoringMetricsServiceEndpointDown","pending","2021-04-16T12:51:43.435642699Z"
============================================================================
============================================================================

```

## Checklist
- [ ] This change requires a documentation update 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added a test case that will be used to verify my changes 
- [ ] Verified independently on a cluster by reviewer

### CODE REVIEW

File: scripts/alerts-during-perf-testing.sh
Comment (trepel) [inline scripts/alerts-during-perf-testing.sh:11]: I think we can get the namespace prefix dynamically, e.g:
https://github.com/integr8ly/integreatly-operator/blob/master/scripts/setup-sso-idp.sh#L8

But given we do perf testing only against RHOAM I think it is optional

File: test-cases/tests/multiAZ/p01-measure-downtime-during-down-az.md
Comment (trepel) [inline test-cases/tests/multiAZ/p01-measure-downtime-during-down-az.md:49]: One liner can be used for this (inspired by Laura): `oc scale dc workload-web-app --replicas=3 -n workload-web-app`

Comment (trepel) [reply test-cases/tests/multiAZ/p01-measure-downtime-during-down-az.md:49]: Also, quite minor (optional)

**Review by trepel — APPROVED** [review]

**Comment by trepel** [comment]
small comments for minor optional improvements, but it is ok to be merged as is. I will lgtm and approve again if you decide to address them (if you managed before this gets merged).

/lgtm
/approve

File: scripts/alerts-during-perf-testing.sh
Comment (austincunningham) [reply scripts/alerts-during-perf-testing.sh:11]: :+1:

File: test-cases/tests/multiAZ/p01-measure-downtime-during-down-az.md
Comment (austincunningham) [reply test-cases/tests/multiAZ/p01-measure-downtime-during-down-az.md:49]: :+1:

**Comment by austincunningham** [comment]
@trepel if you can stick another lgtm on this again

**Pull request merged**
